Rakefile

```diff
@@ -1,4 +1,6 @@
+require 'bundler/gem_tasks'
```

Comment:
If it's useful (and we have to use Bundler anyway), then I'm fine with keeping it.